utils.c

```diff
@@ -212,14 +223,9 @@ int posix_fadvise(int fd, off_t offset, off_t len, int advice)
 	}
 }
 
-#include <unistd.h> /* For usleep(). */
-
-void msleep(double wait_ms)
-{
-	assert(!usleep(wait_ms * 1000));
-}
+#else	/* Apple Macintosh */
 
-#else
+#ifndef __OpenBSD__
 
 #include <time.h> /* For clock_gettime() and clock_nanosleep(). */
 
```

Comment:
Please move this else clause from `#if __APPLE__ && __MACH__` to the new `#if (__APPLE__ && __MACH__) || defined(__OpenBSD__)`. Thus, all versions of `msleep()` are grouped under the same `#if`.

Reply:
Once the change has been made, `git commit --amend` the second (last) commit; there's no need to store the history of changes of this pull request. Once the second commit is updated in your local repository, just `git push --force`.